src/main/ts/components/Editor.tsx

```diff
@@ -30,9 +22,10 @@ export interface IProps {
   toolbar: NonNullable<EditorOptions['toolbar']>;
   disabled: boolean;
   textareaName: string;
-  tinymceScriptSrc: string;
+  tinymceScriptSrc: string | string[] | ScriptItem[];
```

Comment:
From string[] to ScriptItem[] doesn't feel that much different.

Reply:
It allows specifying `async` and `defer` independently which is what I wanted however I didn't want to force people to do that if they didn't need either `async` or `defer`.